transforms/contains-to-includes/index.js

```diff
@@ -0,0 +1,21 @@
+const { getParser } = require('codemod-cli').jscodeshift;
+
+module.exports = function transformer(file, api) {
+  const j = getParser(api);
+
+  return j(file.source)
```

Comment:
Should this `codemod` assure the variable is of type `Array` or `Enumerable` or we can assure the input will always be with variables which is kind is one of these 2.

Here's an example of input/output which I'm not sure if it should be covered:

### **expected diff input-output**
```diff
const myAwesomeObj = {};
myAwesomeObj.contains = (param) => console.log(`contains ${param}`);

myAwesomeObj.contains(1);
```

### **actual diff input-output**
```diff
const myAwesomeObj = {};
myAwesomeObj.contains = (param) => console.log(`contains ${param}`);

- myAwesomeObj.contains(1);
+ myAwesomeObj.includes(1);

```

Reply:
Because this isn't run at runtime and just looks at the code, i dont believe we can actually know the type of a variable, i could be wrong but i'm not familiar on how that would be done.

Reply:
Yep... I guess it depends on how Ember is implementing `Enumerable`... Do this snippet work for Arrays or there would be use-cases left behind with this check?

```js
["my_variable_name"].constructor.name === "Array"
```

Reply:
so in the case where it is just a variable all that the parser would see it an object of tyep `identifier` for example 
```js
thing = "hello";
```
would just show is :

Reply:
I see... maybe with other parsers? Let's move forward. If a user faces this issue we can re-open this research.